src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
It makes me double-take every time I see tox output something like the following, when I'm really on the develop branch:

```
py36 inst-nodeps: C:\<snip>\paho_python\.tox\dist\paho-mqtt-1.2.2.zip
```